plugins/hydra_ray_launcher/tools/get_ray_yaml.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import os
+import tempfile
+
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+
+@hydra.main(config_path="config.yaml")
+def save_yaml(cfg: DictConfig) -> None:
```

Comment:
function name is not great. I think either main() or get_ray_yaml() would be better.

Reply:
Deleted this per our previous discussion. Created a task to look into how do we provide a better way for users to connect to a cluster.